test/known_issues/test-fs-error-no-close.js

```diff
@@ -23,37 +23,16 @@ const tmpdir = require('../common/tmpdir');
 tmpdir.refresh();
 
 {
-  // debuglog('start test');
-
-  // function WriteStream(...args) {
-  //   debuglog('WriteStream constructor');
-  //   fs.WriteStream.call(this, ...args);
-  // }
-  // Object.setPrototypeOf(WriteStream.prototype, fs.WriteStream.prototype);
-  // Object.setPrototypeOf(WriteStream, fs.WriteStream);
-
-  // WriteStream.prototype.open = common.mustCall(function WriteStream$open() {
-  //   debuglog('WriteStream open() callback');
-  //   const that = this;
-  //   fs.open(that.path, that.flags, that.mode, (err, fd) => {
-  //     debuglog('inner fs open() callback');
-  //     that.emit('error', err);
-  //   });
-  // });
-
   fs.open(`${tmpdir.path}/dummy`, 'wx+', common.mustCall((err, fd) => {
     debuglog('fs open() callback');
     assert.ifError(err);
-    // const w = new WriteStream(`${tmpdir.path}/dummy`,
-    //                           { flags: 'wx+', emitClose: true })
-    //   .on('error', common.mustCall((err) => {
-    //     debuglog('error event callback');
-    //     assert.strictEqual(err.code, 'EEXIST');
-    //     w.destroy();
-    //     w.on('close', common.mustCall(() => {
-    //       debuglog('close event callback');
-    //     }));
-    //   }));
+    // The tests in known_issues are not interpreted as "failed as expected"
+    // when they time out, so we need this timer. It can be removed if/when the
+    // issue is fixed and this test is moved out of known_issues.
```

Comment:
I think you can add `TIMEOUT` to the `*.status` file as an alternative to adding this timer.

https://github.com/nodejs/node/blob/1ce906c16ba09885fb372962a3a56ce64ef06a20/tools/test.py#L935

https://github.com/nodejs/node/blob/1ce906c16ba09885fb372962a3a56ce64ef06a20/tools/test.py#L589-L598